.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -192,19 +194,19 @@ jobs:
         uses: azure/CLI@v1
         with:
           inlineScript: |
-            az vm create -g Testnet -n "${{vars.RESOURCE_STARTING_NAME}}-${{ matrix.host_id }}-${{ GITHUB.RUN_NUMBER }}" \
+            az vm create -g Testnet -n "${{ vars.AZURE_HOSTNAME_PREFIX }}-${{ matrix.host_id }}-${{ GITHUB.RUN_NUMBER }}" \
             --admin-username obscurouser --admin-password "${{ secrets.OBSCURO_NODE_VM_PWD }}" \
-            --public-ip-address-dns-name "obscuronode-${{ matrix.host_id }}-${{vars.RESOURCE_TESTNET_NAME}}-${{ GITHUB.RUN_NUMBER }}" \
-            --tags deploygroup=ObscuroNode-${{vars.RESOURCE_TESTNET_NAME}}-${{ GITHUB.RUN_NUMBER }}  ${{vars.RESOURCE_TAG_NAME}}=true \
-            --vnet-name ${{vars.RESOURCE_TESTNET_NAME}}-virtual-network --subnet ${{vars.RESOURCE_TESTNET_NAME}}-sub-network \
+            --public-ip-address-dns-name "obscuronode-${{ matrix.host_id }}-${{ vars.AZURE_RESOURCE_NAME_L2 }}-${{ GITHUB.RUN_NUMBER }}" \
+            --tags deploygroup=ObscuroNode-${{ vars.AZURE_RESOURCE_NAME_L2 }}-${{ GITHUB.RUN_NUMBER }}  ${{ vars.AZURE_RESOURCE_TAG_L2 }}=true \
+            --vnet-name ${{ vars.AZURE_RESOURCE_NAME_L2 }}-virtual-network --subnet ${{ vars.AZURE_RESOURCE_NAME_L2 }}-sub-network \
```

Comment:
is `AZURE_RESOURCE_NAME_L2` the testnet type name ? Can the name be modified to something like `TESTNET_TYPE` ?

Reply:
Can I just use `github.event.inputs.testnet_type` or do you think it's important so duplicate into a var for futureproofing? If it's a var then maybe `TESTNAME_NAME` or `TESTNET_ENVIRONMENT`?

Reply:
Yeah good idea 👍